_data/trans/hi.json

```diff
@@ -0,0 +1,47 @@
+{
+    "about": "हमारे बारे मे",
+    "contribute": "GitHub पर योगदान दे |",
+    "defaultnote_easy": "दिए हुए पेज पर जाए और अपना अकाउंट मिटाए |",
+    "defaultnote_email": "दिए गए पते पर अकाउंट मिटाने के लिए ईमेल करे |",
+    "difficulty": "स्तर",
+    "difficulty_easy": "आसान",
+    "difficulty_hard": "कठिन",
+    "difficulty_impossible": "असंभव",
+    "difficulty_limited": "बहुत मुश्किल",
+    "difficulty_medium": "मध्यम",
+    "extension_browser": "क्रोम या फायर फॉक्स एक्सटेंशन",
+    "extensionguide": "Extesnion के बिंदुओं का मतलब",
```

Comment:
@shivam-maurya-git should this be "Extension", like below?

Reply:
Yes, it is better. Hindi Speakers understand this much English ( Technical Words). Hindi translation of  "extension" will be hard word for readers.